src/logger.js

```diff
@@ -0,0 +1,27 @@
+import {createLogger, format, transports} from 'winston';
+
+const options = {
+  format: format.combine(
+    format.colorize(),
+    format.simple(),
+  ),
+  silent: process.env.NODE_ENV === 'production',
+};
+
+const consoleTransport = new transports.Console(options);
+let fileTransport;
+
+const transportOptions = [consoleTransport];
+
+if (typeof window === 'undefined') {
+  fileTransport = new transports.File({
+    filename: 'logger.log',
```

Comment:
better use an environment variable set in .env: `WEBEX_LOG_FILE`. If and only if set, add the `File` transport